server/sonar-web/src/main/js/components/common/action-options-view.js

```diff
@@ -81,22 +81,22 @@ export default PopupView.extend({
     const that = this;
     this.currentKeyScope = key.getScope();
     key.setScope(this.keyScope);
-    key('down', this.keyScope, function () {
+    key('down', this.keyScope, () => {
       return that.selectNextOption();
     });
-    key('up', this.keyScope, function () {
+    key('up', this.keyScope, () => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)